src/main/java/ScenarioReader.java

```diff
@@ -16,26 +13,30 @@
     private OWLOntologyManager ontologyManager;
     private OWLOntology ontology;
     private MyFactory factory;
-    public static final String name = "http://www.w3.org/2003/11/";
+    public static final String INITIAL_NAME = "http://www.w3.org/2003/11/";
```

Comment:
dalej nie mam pojęcia czyje to `name` :P

Reply:
Coś w stylu IRI_PREFIX wyczerpuje temat :)

Reply:
A może INDIVIDUAL_PREFIX?